ansibullbot/utils/moduletools.py

```diff
@@ -80,6 +118,16 @@ def manage_checkout(self):
             changed = self.update_checkout()
         return changed
 
+    def get_files(self):
+
+        cmd = 'find %s' % self.checkoutdir
```

Comment:
`self.files` should not contains directories (nor `.pyc`). Should not `git ls-files` used instead ?